api/TrackHandler.py

```diff
@@ -94,7 +94,7 @@ def addLabel(data):
 
 # Removes label from label file
 def removeLabel(data):
-    rel_path = '%s%s_Labels.bedGraph' % (cfg.dataPath, data['name'])
+    rel_path = os.path.join(cfg.dataPath, data['name'] + '_Labels.bedGraph')
```

Comment:
this logic is repeated in addLabel funciton above. please avoid repetition, for example by creating a class with a method get_bedGraph_file which you can call from both removeLabel and addLabel

Reply:
again please consider using/adapting the class system defined in https://github.com/tdhock/SegAnnDB/blob/master/plotter/db.py where each class represents a berkeley db database and each instance represents an entry in that database